v20.2/upgrade-cockroach-version.md

```diff
@@ -70,14 +64,30 @@ By default, after all nodes are running the new version, the upgrade process wil
 
     {% include copy-clipboard.html %}
     ~~~ sql
-    > SET CLUSTER SETTING cluster.preserve_downgrade_option = '20.2';
+    > SET CLUSTER SETTING cluster.preserve_downgrade_option = '20.1';
     ~~~
 
     It is only possible to set this setting to the current cluster version.
 
 ### Features that require upgrade finalization
 
-This information is TBD pending further development of CockroachDB v20.2.
+When upgrading from v20.1 to v20.2, certain features and performance improvements will be enabled only after finalizing the upgrade, including but not limited to:
```

Comment:
@jordanlewis, @knz, here's where we mentioned features that require 20.2 upgrade finalization before they can be used. Can either of you review or tell me who to ping? 

Also, I didn’t explicitly mention some migrations that don’t seem user-visible, but I’d like someone to confirm that these don’t need to be mentioned: https://github.com/cockroachdb/docs/issues/8627#issuecomment-716076611

Reply:
This LGTM. I think the approach of looking at these version numbers makes sense.

The other migrations seem fine, except for maybe these two:

@knz, does VersionHBAForNonTLS need documentation?

@dt, do we need to document the change to the scheduled jobs table that drops `schedule_changes` and adds `schedule_status`?

Reply:
the scheduled jobs table was introduced in 20.2 development so don't think we need to discuss migrations on it.